README.md

```diff
@@ -41,6 +41,10 @@ android {
     }
 }
 ```
+add google_cerver_client meta-data in application tag on your manifest file to get idToken for backend authentication
```

Comment:
Right above this line, you can see the manifest placeholders I have created for any values that need to be in the manifest. With this ```google_server_client_id```, we should take the same approach, both for ease of use and consistency.
https://github.com/iFanie/Roguin/blob/c7619ff470f9b56f4058d281005c07522f5f826e/roguin/src/main/AndroidManifest.xml#L28

See the above placeholder and follow the same approach. After that, the end user would just need to add another placeholder in their build file.

Reply:
yep. thats nice